Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1281,6 +1292,51 @@ def timestamp_length_equalization(timestamp1, timestamp2):
     return int(timestamp1), int(timestamp2)
 
 
+def change_host_group(method: str,
+                      host_group_id: str = None,
+                      name: str = None,
+                      group_type: str = None,
+                      description: str = None,
+                      assignment_rule: str = None) -> CommandResults:
+    allowed_methods = {'POST', 'PATCH'}
+    if method not in allowed_methods:
+        raise DemistoException(f'CrowdStrike Falcon error: method should be in {allowed_methods}')
+
+    data = {'resources': [{
+        'id': host_group_id,
+        "name": name,
+        "description": description,
+        "group_type": group_type,
+        "assignment_rule": assignment_rule
+    }]}
+    response = http_request(method=method,
+                            url_suffix='/devices/entities/host-groups/v1',
+                            json=data)
+    resources = response.get('resources')
+    return CommandResults(outputs_prefix='CrowdStrike.HostGroup',
+                          outputs_key_field='id',
+                          outputs=resources)
+
+
+def change_host_group_members(action_name: str,
+                              host_group_id: str,
+                              host_ids: List[str]) -> CommandResults:
+    allowed_actions = {'add-hosts', 'remove-hosts'}
+    if action_name not in allowed_actions:
+        raise DemistoException(f'CrowdStrike Falcon error: action name should be in {allowed_actions}')
+    data = {'action_parameters': [{'name': 'filter',
+                                   'value': f"(device_id:{str(host_ids)})"}],
+            'ids': [host_group_id]}
+    response = http_request(method='POST',
+                            url_suffix='/devices/entities/host-group-actions/v1',
+                            params={'action_name': action_name},
+                            json=data)
+    resources = response.get('resources')
+    return CommandResults(outputs_prefix='CrowdStrike.HostGroup',
+                          outputs_key_field='id',
+                          outputs=resources)
```

Comment:
Please pass the response as part of the raw_response.
Also, note that the _command should create the CommandResults given the response.